### PR TITLE
Update links of cipher suites restriction

### PR DESCRIPTION
**Pull request by cicku**

### CODE REVIEW

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md [hunk 1]
```diff
@@ -24,21 +24,19 @@ ___
 
 ## Solution
 
-If the Universal SSL _does not meet your business requirements_, we would recommend using our _Advanced Certificate Manager_. For example, use the Advanced Certificate Manager to cover more than one level of subdomain, remove Cloudflare branding from the Universal certificate, or adjust the shortest certificate lifespan.
+Cipher suites has to be disabled when you have [Advanced Certificate Manager](/ssl/edge-certificates/advanced-certificate-manager) purchased. After you subscribed to Advance Certificate Manager for your domain, you can [restrict Cipher Suites at the Zone-level requests via the API](/api/operations/zone-settings-change-ciphers-setting). Currently, restricting cipher suites could only be done via API and not available via Cloudflare Dashboard. Assuming this is just a one-time change, you could trigger the API call using curl. **Step-by-step guide:**
```
Comment (ngayerie) [inline content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md]: I assume you meant "Cipher suites can only be disabled" here?

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md [hunk 2]
```diff
@@ -24,21 +24,19 @@ ___
...
 
-You could restrict the Cipher Suites used for TLS using our [Advanced Certificate Manager](/ssl/edge-certificates/advanced-certificate-manager). After you subscribed to Advance Certificate Manager for your domain, you can [restrict Cipher Suites at the Zone-level requests via the API](/api/operations/zone-settings-change-ciphers-setting). Currently, restricting cipher suites could only be done via API and not available via Cloudflare Dashboard. Assuming this is just a one-time change, you could trigger the API call using curl. **Step-by-step guide:**
-
-1.  Get/view Global API Key _(or create Token)_ from: [https://dash.cloudflare.com/profile/api-tokens](https://dash.cloudflare.com/profile/api-tokens)
-2.  Get Zone ID from the bottom right of Overview page for your domain in Cloudflare Dashboard.
-3.  Decide which Cipher Suites you would like to allow from [the list](/ssl/reference/cipher-suites/).
+1.  [Get/view Global API Key _(or create Token)_](/fundamentals/api/get-started/keys/)
```
Comment (ngayerie) [inline content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md:29]: The link to the Dashboard is correct, I think it makes sense to keep it?

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md (hunk 1 above)
Comment (ngayerie) [reply content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md]: Advance Certificate Manager
-> 
Advanced Certificate Manager

**Review by ngayerie — CHANGES_REQUESTED** [review]

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md (hunk 2 above)
Comment (cicku) [reply content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md:29]: Doc refs always have higher priority than actual URLs.

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md (hunk 1 above)
Comment (cicku) [reply content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md]: Fixed

File: content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md [hunk 3]
```diff
@@ -24,21 +24,19 @@ ___
 
 ## Solution
 
-If the Universal SSL _does not meet your business requirements_, we would recommend using our _Advanced Certificate Manager_. For example, use the Advanced Certificate Manager to cover more than one level of subdomain, remove Cloudflare branding from the Universal certificate, or adjust the shortest certificate lifespan.
+Cipher suites has to be disabled when you have [Advanced Certificate Manager](/ssl/edge-certificates/advanced-certificate-manager) purchased. After you have subscribed to Advanced Certificate Manager for the zone, you can [restrict Cipher Suites at the Zone-level requests via the API](/api/operations/zone-settings-change-ciphers-setting). Currently, restricting cipher suites could only be done via API and not available via Cloudflare Dashboard. Assuming this is just a one-time change, you could trigger the API call using curl. **Step-by-step guide:**
```
Comment (ngayerie) [inline content/support/SSL TLS/Edge Certificates SSL TLS configuration/Disabling Weak Cipher Suites.md:27]: "Cipher suites has to be disabled when you" 
This needs to be updated

**Review by ngayerie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**